### PR TITLE
John conroy/organs link homepage

### PR DESCRIPTION
**Pull request by john-conroy**
Screen Shot 2022-08-26 at 12 02 20 PM

### CODE REVIEW

File: context/app/routes_main.py [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 
 
 blueprint = make_blueprint(__name__)
 
 
 @blueprint.route('/')
 def index():
-    flask_data = {**get_default_flask_data()}
+    organs = get_organs()
+    flask_data = {**get_default_flask_data(), 'organs': organs}
```
Comment (mccalluc) [inline context/app/routes_main.py:13]:
```suggestion
    flask_data = {**get_default_flask_data(), 'organ_count': len(get_organs())}
```

Or something like that. See comment below.

**Review by mccalluc — COMMENTED** [review]
Slight preference to only pass the information needed, but not a big deal if you feel differently.

File: context/app/static/js/components/home/EntityCounts/EntityCounts.jsx [hunk 2]
```diff
@@ -25,33 +20,32 @@ const entities = [
...
+            icon={<StyledSvgIcon component={icon} color="primary" />}
             count={entityCounts?.[entity_type]}
             label={`${entity_type}s`}
             href={`/search?entity_type[0]=${entity_type}`}
           />
         ))}
-        <EntityCount icon={CollectionIcon} count={entityCounts?.Collection} label="Collections" href="/collections" />
+        <EntityCount
+          icon={<StyledURLSvgIcon iconURL={organs?.kidney.icon} ariaLabel="Organs Icon" />}
+          count={Object.keys(organs).length}
```
[the remark this answers is not in the document]
Comment (john-conroy) [reply context/app/static/js/components/home/EntityCounts/EntityCounts.jsx:39]: We need the icon for the kidney.

File: context/app/routes_main.py (hunk 1 above)
Comment (john-conroy) [reply context/app/routes_main.py:13]: Initially I went in this direction, but we need the icon for the kidney. I could hard code it?

File: context/app/static/js/components/home/EntityCounts/hooks.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+function useEntityCounts() {
+  const [entityCounts, setEntityCountsData] = useState(undefined);
+  const { searchData: elasticsearchData } = useSearchData(entityCountsQuery);
+  useEffect(() => {
+    if (Object.keys(elasticsearchData).length) {
+      const entityCountsObject = elasticsearchData.aggregations.entity_type.buckets.reduce((acc, entity) => {
+        const accCopy = acc;
+        accCopy[entity.key] = entity.doc_count;
+        return accCopy;
+      }, {});
```
Comment (mccalluc) [inline context/app/static/js/components/home/EntityCounts/hooks.js:19]: Slight preference for named function that explains what's happening.

Comment (john-conroy) [reply context/app/static/js/components/home/EntityCounts/hooks.js:19]: I think this is similar to a python comprehension in its complexity, but I can pull it to a function.

Comment (john-conroy) [reply context/app/static/js/components/home/EntityCounts/hooks.js:19]: It is older code that could be simplified as well.

**Pull request merged**